ui/app/css/itcss/components/item.scss

```diff
@@ -0,0 +1,116 @@
+.Item__flex-grid {
```

Comment:
Nit: most class names in the code-base are not capitalized. Lower case `I` for consistency?